pom.xml

```diff
@@ -286,22 +264,31 @@
     </build>
 
     <dependencies>
+<!--        Note that we are using 2.3.z version under jakarta groupId, which still uses the javax. package name in code.-->
         <dependency>
-            <groupId>javax.servlet.jsp</groupId>
-            <artifactId>javax.servlet.jsp-api</artifactId>
+            <groupId>jakarta.servlet.jsp</groupId>
+            <artifactId>jakarta.servlet.jsp-api</artifactId>
             <version>${jsp.api.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
             <version>${servlet.api.version}</version>
             <scope>provided</scope>
         </dependency>
+<!--        Needed to make Tomcat log in tests-->
```

Comment:
What is used for hazelcast logs? wouldn't it be better to have single logging library where you could configure both tomcat and hazelcast (log4j 2.x)

Reply:
We disable hazelcast logging via properties as you can see, but if it's not set, log4j is used. https://github.com/hazelcast/hazelcast-wm/blob/09b8b5284087caf13be2e4e5d9d5ef0cbad442a9/src/test/java/com/hazelcast/wm/test/AbstractWebFilterTest.java#L67

Well, I simply wanted to use the easiest approach. We are not configuring the tomcat logging or anything. For some reason, it wasn't logging with some warning message like "SL4J no-op logger is used". After researching about it, the simplest solution to make it log was this one. 

I have no idea how to configure tomcat to use log4j though.

Reply:
IIUC tomcat uses slf4j, so you need to use the correct slf4j implementation. Instead of this simple one use

```
        <dependency>
            <groupId>org.apache.logging.log4j</groupId>
            <artifactId>log4j-slf4j2-impl</artifactId>
            <scope>test</scope>
        </dependency>
```
and it should work fine, and this pulls in log4j-core, which can be used for hazelcast logging.

I have no idea why Hazelcast logs are turned off, when I was fixing the issue with connection timeout I had to remove that to figure out what is the issue.

Reply:
> I have no idea why Hazelcast logs are turned off, when I was fixing the issue with connection timeout I had to remove that to figure out what is the issue.

well i don't have no idea as well, it was that way. let me remove that actually.

> IIUC tomcat uses slf4j, so you need to use the correct slf4j implementation. Instead of this simple one use

trying